meshnets/data_processing/data_mappers.py

```diff
@@ -32,3 +32,25 @@ def to_undirected(example):
     edges = np.concatenate([edges, edges[:, ::-1]], axis=0)
     example['edges'] = edges
     return example
+
+
+def make_edge_features(example):
+    """Makes the edge features from the node coordinates.
+
+    The edge features:
+
+    - The displacement vector between the source and target nodes.
+    - The distance between the source and target nodes.
+
+    """
+    nodes = np.array(example['nodes'])
+    edges = np.array(example['edges'])
+
+    if edges.size == 0 or nodes.size == 0:
+        example['edge_features'] = []
```

Comment:
Maybe I am missing something... But why I call the input argument example?

Reply:
You are right. I should have given you more context. These functions are to be used to map over our Hugging face datasets. For example:

```python
dataset = datasets.load('wind-tunnel-dataset')
dataset = dataset.map(make_edge_features)
```

This will convert the raw data into whichever format we may want to use to solve the problem.

I called the function argument `example` to follow HuggingFace's [example](https://huggingface.co/docs/datasets/process#map) on mapping over data